### ISSUE TITLE
Вопрос по 3-ей лабе, 19-ый вариант

### ISSUE THREAD

**Issue by ValeriyaSinevich** [agsh/mipt2016#12]
Для каждого языка программирования я хочу найти его создателя. Можно ограничиться взятием значения в "Designed by" в табличке справа (в случае ее присутствия)? например, у C это Dennis Ritchie: https://en.wikipedia.org/wiki/C_(programming_language).
В случае отсутствия таблички или отсутствия в ней поля "Designed by" я говорю, что про язык ничего неизвестно. 
Нормально так?

**Comment by agsh** [comment]
Да, можно завязаться на одно это поле, а результат обернуть в option
